js/scripts.js

```diff
@@ -247,3 +247,35 @@ function addPortfolioWork() {
 
 // Called  the Function to fill the PortfolioElement.
 addPortfolioWork();
+
+// FORM VALIDATION
+// function to convert the content of the email field  to be in lower case
+function convertToLower(formType) {
+  formType.onkeyup = (() => {
+    formType.value = formType.value.toLowerCase();
+    return formType.value;
+  });
+}
+
+const emailInput = document.getElementById('email');
+const errorMessage = document.getElementById('error-message');
+
+convertToLower(emailInput);
```

Comment:
- [ ] Kindly consider checking if the email is all lowercase by comparing if the email and its lowercase converted version is the same,
` ie: email === email.toLowerCase()`, utilizing regular expressions, or looping through each letter of the email string to make your error message be displayed to the user near the submit button informing them of the error and that the form is not sent. cause currently there is no possible way that the error message might be displayed because as i type a capital letter is immediately converted into a small letter and that is not meeting the requirements 😀